docs/processors/builtin.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: 'Built-in processors'
+sidebar_position: 1
+---
+
+:::note
+Note that all built-in processors that operate on the payload actually operate on `Record.Payload.After`.
+If you need to manipulate the field `Record.Payload.Before` you can use
+a [JavaScript processor](./javascript-processors).
+:::
+
+Below is a list of available built-in processors in Conduit. An example usage of a built-in processor is available in
+[`extract-field-transform.sh`](https:github.com/ConduitIO/conduit/blob/main/examples/processors/extract-field-transform.sh).
```

Comment:
The link seems to be broken. Although, I'd rather point to the [configuration file](https://github.com/ConduitIO/conduit/blob/main/examples/processors/pipeline-extract-field-transform.yml) instead of the shell script. Or we just link to the [folder](https://github.com/ConduitIO/conduit/tree/main/examples/processors) and ideally add a readme in there that describes the example (essentially we would just copy over the comments from the shell script).

Reply:
I like the idea of linking to the config. file directly a bit more, since we may add more examples to the folder.